tests/lib/sprintf/prj.conf

```diff
@@ -1,4 +1,6 @@
 CONFIG_ZTEST=y
+CONFIG_ZTEST_NEW_API=y
 CONFIG_FPU=y
 CONFIG_TEST_USERSPACE=y
 CONFIG_ZTEST_FATAL_HOOK=y
+CONFIG_STDOUT_CONSOLE=y
```

Comment:
Do we need to add this? It seems like being a little bit different with the original test.

Reply:
Yes, i was debugging something else and forgot to remove this before pushing the code. Removed it now.